appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js

```diff
@@ -0,0 +1,35 @@
+import { parseHtml } from '@pins/platform';
+import nock from 'nock';
+import supertest from 'supertest';
+import { createTestEnvironment } from '#testing/index.js';
+import { appealData } from '#testing/appeals/appeals.js';
+
+const { app, teardown } = createTestEnvironment();
+const request = supertest(app);
+const baseUrl = '/appeals-service/appeal-details';
+
+describe('start-case', () => {
+	beforeEach(() => {
+		nock('http://test/').get('/appeals/1').reply(200, appealData);
+	});
+	afterEach(teardown);
+
+	describe('GET /start-case/date', () => {
+		it('"Start case" page should contain "Confirming will activate the timetable on"', async () => {
+			const response = await request.get(`${baseUrl}/1/start-case/date`);
+			expect(response.text).toContain('Confirming will activate the timetable on');
+		});
+
+		it('should redirect to the confirmation page', async () => {
+			nock('http://test/').patch(`/appeals/1`).reply(200, {});
+			const response = await request.post(`${baseUrl}/1/start-case/date`).send({});
+			expect(response.statusCode).toBe(302);
+		});
+
+		it('should render "Case started" confirmation page should', async () => {
```

Comment:
This should be moved into a separate `describe` - this one is just for `GET /start-case/date`. Same for the `POST` test above.

Reply:
done